dom/dom_c.php

```diff
@@ -179,35 +180,38 @@ public function insertBefore(
     /**
      * Replaces a child
      * @link https://php.net/manual/en/domnode.replacechild.php
+     * @template TNode of DOMNode
      * @param DOMNode $node <p>
      * The new node. It must be a member of the target document, i.e.
      * created by one of the DOMDocument->createXXX() methods or imported in
      * the document by .
      * </p>
-     * @param DOMNode $child <p>
+     * @param TNode $child <p>
      * The old node.
      * </p>
-     * @return DOMNode|false The old node or false if an error occur.
+     * @return TNode|false The old node or false if an error occur.
```

Comment:
According to the phpdoc and https://www.php.net/manual/en/domnode.replacechild.php
the old node is returned which is the second param if I understand.